bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/thing/lights.xml

```diff
@@ -32,6 +32,21 @@
 		<config-description-ref uri="thing-type:shelly:light"/>
 	</thing-type>
 
+	<thing-type id="shellycolorbulb">
+		<label>Shelly Duo Color Bulb (SHSCB-1)</label>
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
but this is the product name

Reply:
You could remove the abbreviation, or?

Reply:
all other thing labels are following this schema

Reply:
There will be a checkstyle rule soon, which warns at 20 chars and generates an error at 25 chars. Would be great if you could re-work the labels in a follow-up PR.